### DISCUSSION TITLE
Amazing work!

### DISCUSSION THREAD

**Discussion by yetti7** [mikiher/audiobookshelf-windows#16] [General]
Thank you so much for this. Im bound by current systems not being able to use Docker and this has made my life 100x easier. I look forward to the 2.10.1 release to fix the transcode issues!

**Comment by petenun** [comment]
I second this sentiment. Thank you @mikiher! You have improved my life considerably by offering this Windows version. I realize that most devs on Github code as a labor of love, but it would be nice to give back and show appreciation. Do you accept Paypal or Venmo donations? I'd at least like to buy you a couple beers, coffees, or whatever you're into, as appreciation.

**Reply by mikiher** [reply]
Thanks, much appreciated! 

If you'd like to donate, please donate to @advplyr, our parent project owner, at https://github.com/sponsors/advplyr - this is the donations page for all the Audiobookshelf umbrella of projects.
